scripts/jenkins/ardana/manual/lib.sh

```diff
@@ -313,3 +313,7 @@ function exit_msg {
 *****************************************************************************************
     "
 }
+
+# Local Variables:
```

Comment:
I'd prefer one `.dir-locals.el` for the whole repo.

Reply:
Ah nice! I like that better too, thanks! :smiley: